dev-qt/qtconcurrent/qtconcurrent-5.14.9999.ebuild

```diff
@@ -14,7 +14,7 @@ fi
 IUSE=""
 
 DEPEND="
-	~dev-qt/qtcore-${PV}
+	~dev-qt/qtcore-${PV}:${SLOT/\/*}=
```

Comment:
is this equivalent to `${SLOT%/*}`? or we could even hardcode `:5=` I guess

Reply:
it is, in fact, yes.

I was actually wondering why the dependencies stopped short of the slot def already.

In the end, when these ebuilds are going to be ported to Qt6 another run to /dev-qt/s/5/6/ won't matter so I'll just convert to `:5=`.

Reply:
> I was actually wondering why the dependencies stopped short of the slot def already.

No reason I guess. A `:5` SLOT dep is probably redundant in a `~${PV}` style dep. I guess that was the reason.

> In the end, when these ebuilds are going to be ported to Qt6 another run to /dev-qt/s/5/6/ won't matter so I'll just convert to `:5=`.

Sure, I'm fine either way.